src/de/error.rs

```diff
@@ -78,6 +82,8 @@ impl StdError for Error {
                 ParseError::Eof => "Unexpected end of file",
                 ParseError::ExpectedArray => "Expected array",
                 ParseError::ExpectedArrayEnd => "Expected end of array",
+                ParseError::ExpectedAttribute => "Expected an enable attribute",
+                ParseError::ExpectedAttributeEnd => "Expected a closing `]` after the attribute",
```

Comment:
Ah, this could also fail if `)` is missing. The error message should reflect that.